src/solvers/refinement/string_constraint_generator_main.cpp

```diff
@@ -247,16 +244,35 @@ string_constraint_generatort::associate_char_array_to_char_pointer(
   array_string_exprt array_sym =
     to_array_string_expr(fresh_symbol(symbol_name, char_array_type));
   auto insert_result =
-    arrays_of_pointers_.insert(std::make_pair(char_pointer, array_sym));
+    arrays_of_pointers.insert(std::make_pair(char_pointer, array_sym));
   array_string_exprt result = to_array_string_expr(insert_result.first->second);
+#if 0
```

Comment:
Please add a comment why this is deactivated

Reply:
actually I'm going to remove this line, it cannot be activated, and is unnecessary (default axioms are added on fresh_strings, for the already existing strings, constraints are unnecessary)